modules/core/src/com/haulmont/addon/ldap/core/service/MatchingRuleAccessGroupServiceBean.java

```diff
@@ -0,0 +1,25 @@
+package com.haulmont.addon.ldap.core.service;
+
+import com.google.common.base.Strings;
+import com.haulmont.addon.ldap.entity.AbstractDbStoredMatchingRule;
+import com.haulmont.addon.ldap.service.MatchingRuleAccessGroupService;
+import com.haulmont.cuba.security.entity.Group;
+import com.haulmont.cuba.security.group.AccessGroupsService;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Inject;
+
+@Service(MatchingRuleAccessGroupService.NAME)
+public class MatchingRuleAccessGroupServiceBean implements MatchingRuleAccessGroupService {
+
+	@Inject
+	private AccessGroupsService accessGroupsService;
+
+	@Override
+	public Group getAccessGroupMatchingRule(AbstractDbStoredMatchingRule rule) {
```

Comment:
The name `getAccessGroupMatchingRule` looks like the method is going to return a matching rule. Maybe we can use something like `getAccessGroupForMatchingRule`